.reuse/dep5

```diff
@@ -0,0 +1,12 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: cloud-hypervisor
+Upstream-Contact:  <>
+Source: https://www.cloudhypervisor.org
+
+Files: docs/*.md *.md
+Copyright: 2024
+License: CC-BY-4.0
```

Comment:
Since the `CC-BY-4.0.txt` license file is moved to a different location. The opening statement should be adjusted.
https://github.com/cloud-hypervisor/cloud-hypervisor/blob/493178c97254f3e3b1e5e6e5083a53982704ff83/docs/COPYING.docs#L1
 
We can say something like:

```
All documentations (e.g. files with extension `.md`) in this repository is covered by the following license:
```

Reply:
done